quarterly_report.rb

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env ruby
+#This script is for discovering quarterly metrics of PRs in total. How many are created, closed and merged by community members in a specified quarter.
+
+require 'optparse'
+require 'csv'
+require 'octokit'
+require_relative 'octokit_utils'
+
+options = {}
+options[:oauth] = ENV['GITHUB_COMMUNITY_TOKEN'] if ENV['GITHUB_COMMUNITY_TOKEN']
+parser = OptionParser.new do |opts|
+  opts.banner = 'Usage: quarterly_report.rb [options]'
+
+  opts.on('-n', '--namespace NAME', 'GitHub namespace. Required.') { |v| options[:namespace] = v }
+  opts.on('-r', '--repo-regex REGEX', 'Repository regex') { |v| options[:repo_regex] = v }
+  opts.on('-t', '--oauth-token TOKEN', 'OAuth token. Required.') { |v| options[:oauth] = v }
+  opts.on('-o', '--overview', 'Output overview, summary totals to csv') { options[:display_overview] = true}
+
+  # default filters
+  opts.on('--puppetlabs', 'Select Puppet Labs\' modules') {
+    options[:namespace] = 'puppetlabs'
+    options[:repo_regex] = '^puppetlabs-'
+  }
+  opts.on('--puppetlabs-supported', 'Select only Puppet Labs\' supported modules') {
+    options[:namespace] = 'puppetlabs'
+    options[:repo_regex] = OctokitUtils::SUPPORTED_MODULES_REGEX
+  }
+  opts.on('--community', 'Select community modules') {
+    options[:namespace] = 'puppet-community'
```

Comment:
voxpupuli, not puppet-community; should proabably fixed in all scripts - or, as discussed, pulled into common define